packages/salesforcedx-vscode-core/src/commands/forceStartApexDebugLogging.ts

```diff
@@ -29,32 +32,42 @@ import { developerLogTraceFlag } from '.';
 
 export class ForceStartApexDebugLoggingExecutor extends SfdxCommandletExecutor<{}> {
   public build(): Command {
-    return new SfdxCommandBuilder()
-      .withDescription(nls.localize('force_start_apex_debug_logging'))
-      .withArg('force:data:soql:query')
-      .withFlag(
-        '--query',
-        "SELECT id, logtype, startdate, expirationdate, debuglevelid, debuglevel.apexcode, debuglevel.visualforce FROM TraceFlag WHERE logtype='DEVELOPER_LOG'"
-      )
-      .withArg('--usetoolingapi')
-      .withArg('--json')
-      .build();
+    return new CommandBuilder(
+      nls.localize('force_start_apex_debug_logging')
+    ).build();
+  }
+
+  public attachSubExecution(execution: CommandExecution) {
+    channelService.streamCommandOutput(execution);
+    channelService.showChannelOutput();
   }
 
   public async execute(response: ContinueResponse<{}>): Promise<void> {
     const cancellationTokenSource = new vscode.CancellationTokenSource();
```

Comment:
The same cancellation token will be passed to all the subexecutions so pressing the top level cancel will cancel all of them.